CMakeLists.txt

```diff
@@ -590,22 +590,33 @@ CONFIGURE_PACKAGE_CONFIG_FILE(
         SIMBODY_VISUALIZER_INSTALL_DIR
         SIMBODY_INSTALL_DOXYGENDIR
     )
-INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/SimbodyConfig.cmake
-    DESTINATION ${SIMBODY_CMAKE_DIR})
+INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/SimbodyConfigForInstall.cmake
+    DESTINATION ${SIMBODY_CMAKE_DIR}
+    RENAME SimbodyConfig.cmake)
 
 # Create a file that allows clients to Simbody to ensure they have the version
 # of Simbody they want.
 # Requires CMake 2.8.6.
-# Note: this is actually deprecated in the latest CMake. Eventually,
-# we are to use WRITE_BASIC_PACKAGE_VERSION_FILE instead.
-# Writes a ConfigVersion file for us.
-WRITE_BASIC_CONFIG_VERSION_FILE(
-    ${CMAKE_CURRENT_BINARY_DIR}/cmake/SimbodyConfigVersion.cmake
-    VERSION "${SIMBODY_VERSION}"
-    COMPATIBILITY SameMajorVersion)
-INSTALL(FILES ${CMAKE_CURRENT_BINARY_DIR}/cmake/SimbodyConfigVersion.cmake
-    DESTINATION ${SIMBODY_CMAKE_DIR})
+if(CMAKE_VERSION VERSION_LESS 2.8.8) # remove when older CMake versions are no longer supported
+    include(WriteBasicConfigVersionFile)
+    # Writes a ConfigVersion file for us.
+    WRITE_BASIC_CONFIG_VERSION_FILE(
+        ${CMAKE_CURRENT_BINARY_DIR}/cmake/SimbodyConfigVersion.cmake
+        VERSION "${SIMBODY_VERSION}"
+        COMPATIBILITY SameMajorVersion)
+else()
+    include(CMakePackageConfigHelpers)
+     WRITE_BASIC_PACKAGE_VERSION_FILE(
+        ${CMAKE_CURRENT_BINARY_DIR}/cmake/SimbodyConfigVersion.cmake
+        VERSION "${SIMBODY_VERSION}"
+        COMPATIBILITY SameMajorVersion)
+endif()
```

Comment:
Older CMake versions are no longer supported. Can you leave only the second branch? Also, update the comment above to say "Requires CMake 2.8.8.".